python/requirements.txt

```diff
@@ -58,6 +58,6 @@ starlette
 typer
 fsspec
 pandas>=1.3
-pydantic<2
+pydantic!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.*,!=2.4.*,<3
```

Comment:
>=2.5.0,<3 doesn't work?

Reply:
That works, but we want to allow users to use `pydantic<2` for Serve, so no need to pin it to `2.5.0` explicitly for now.

Reply:
Can you add a comment to this effect?

Reply:
Yeah, I added one.